### DISCUSSION TITLE
Example to Schedule by BusinessHour

### DISCUSSION THREAD

**Discussion by OmarLand** [Extended-Thunder/send-later#594] [Q&A] [answered]
Hello!!! I would like to know an example of how to add a personal function to send the mail to a specific hour using the BusinessHours, i dont get it. Or if it´s possible to make it more visual just to pick: The Day + the hour.., Adding this to a Button..,

**Comment by jikamens** [comment, marked as answer]
Hi Omar!

I'm having a bit of trouble understanding what you mean by "send the mail to a specific hour." Can you elaborate a bit on what your goal is?

**Reply by OmarLand** [reply]
Oh Sry! I just got to manage it while reading a bit more the code! It´s a great tool Jikamens!!! Thank you so much!
In the part of BusinessHours, i only wanted to manage the send of a mail in a specific time.., But it´s done!
